code/game/gamemodes/gameticker.dm

```diff
@@ -465,6 +465,7 @@ var/global/datum/controller/gameticker/ticker
 
 /datum/controller/gameticker/proc/declare_completion()
 
+	nologevent = 1 //end of round murder and shenanigans are legal; there's no need to jam up attack logs past this point.

```

Comment:
I think it would be better to put this in a roundend hook. We have one of those, right?